diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
index f194d7a..1aab882 100644
--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -156,9 +156,6 @@ else()
         target_link_libraries(sdrpp_core PUBLIC ${CORRECT_LIBRARIES})
     endif (NOT USE_INTERNAL_LIBCORRECT)
 
-    if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-        target_link_libraries(sdrpp_core PUBLIC stdc++fs)
-    endif ()
 
 endif ()
 
